src/main/java/uk/gov/companieshouse/api/accounts/controller/CurrentPeriodController.java

```diff
@@ -103,4 +138,22 @@ public ResponseEntity get(HttpServletRequest request) {
         return apiResponseMapper.mapGetResponse(responseObject.getData(), request);
 
     }
+
+    /**
+     * Log {@link AbridgedAccount} validation failure
```

Comment:
AbridgedAccount?, It's a private method no need to add a Javadoc so you can just delete it.